QualityControl/lib/controllers/LayoutController.js

```
@@ -94,14 +94,14 @@ export class LayoutController {
     const { id } = req.params;
 
     try {
-      if (!id) {
+      if (!id.trim()) {
         updateAndSendExpressResponseFromNativeError(res, new InvalidInputError('Missing parameter "id" of layout'));
```

I would `throw new InvalidInputError('Missing parameter "id" of layout'))` as `updateAndSendExpressResponseFromNativeError` is within the catch